sass/variables/_colors.scss

```diff
@@ -92,6 +92,7 @@ $text-highlight: $lime-dark;
 $base-link-color: $sky-blue;
 $hover-link-color: $sky-blue-light;
 $visited-link-color: $purple;
+$alt-link-color: $anchor-blue;
```

Comment:
I think a more clear name for this is `nav-text-color` or something that implies that we only use this in navigation, then we can use it with or in place of `$header-text-color` as well because I think these are two things that could be tied together (i.e. if we decided nav link text was all dark grey, I think both areas would change to dark grey text, not just one or the other, so we want them using the same variables)